app/src/main/java/org/olpc/xorobothash/MainActivity.java

```diff
@@ -101,13 +103,12 @@ private void showDisconnected() {
     }
 
     private String robotImagePath() {
-        return this.getFilesDir() + File.pathSeparator + "this-xos-robot019.png";
+        return this.getFilesDir() + File.pathSeparator + "this-xos-robot.png";
```

Comment:
Rather than use `File.pathSeparator` directly, use the Path class. See http://stackoverflow.com/questions/412380/combine-paths-in-java/412495#412495